.gitignore

```diff
@@ -1,42 +1,16 @@
+# Build outputs
 build/
+dist/
+.next
+out
+.nuxt
+.vuepress/dist
+.svelte-kit
+.docusaurus
```

Comment:
I feel like half of these new .gitignore entries are completely unrelated:
- `.next`
- `.nuxt`
- `.vuepress`
- `.svelte-kit`

just to name a few